### PR TITLE
Fix Dockerfile to install pandoc and compile on top of pgbouncer stable-1.19 and deploy with EKS

### PR DESCRIPTION
**Pull request by SanjeevThurgam**
Fixing the Dockerfile to compile on top of pgbouncer stable-1.19 and deploy with EKS. Also fixing a broken link in the readme. 

Fixing the dockerfile entailed updating the script to clone from a more recent stable branch of pgbouncer, and adding a command to install pandoc. In theory, other pgbouncer changes can easily be run and pushed by just editing the branch appropriately in this script.

_____

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+#  && rm -rf /var/lib/apt/lists/*
 FROM public.ecr.aws/amazonlinux/amazonlinux:2022
 
 RUN yum -y update
+RUN yum install -y wget
 RUN yum install -y tar net-tools curl vim unzip less libevent-devel openssl-devel python-devel libtool git patch make gcc --allowerasing
-RUN git clone https://github.com/pgbouncer/pgbouncer.git --branch "pgbouncer_1_15_0" && \
+RUN wget https://github.com/jgm/pandoc/releases/download/3.1.6.2/pandoc-3.1.6.2-linux-amd64.tar.gz && \
+    tar xvzf ./pandoc-3.1.6.2-linux-amd64.tar.gz --strip-components 1 -C /usr/local
+RUN git clone https://github.com/pgbouncer/pgbouncer.git --branch "stable-1.19" && \
```
[the remark this answers is not in the document]
Comment (gitstashpop) [reply Dockerfile:8]: Yep, that's the right branch. see this [commit](https://github.com/awslabs/pgbouncer-rr-patch/commit/a355c11263e1e96b97f58983a77f0e19b981d353)

File: Dockerfile [hunk 2]
```diff
@@ -1,8 +1,11 @@
 FROM public.ecr.aws/amazonlinux/amazonlinux:2022
 
 RUN yum -y update
+RUN yum install -y wget
```
Comment (gitstashpop) [inline Dockerfile:4]: can you add this as part of the line below?

File: Dockerfile [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
 RUN yum install -y tar net-tools curl vim unzip less libevent-devel openssl-devel python-devel libtool git patch make gcc --allowerasing
-RUN git clone https://github.com/pgbouncer/pgbouncer.git --branch "pgbouncer_1_15_0" && \
+RUN wget https://github.com/jgm/pandoc/releases/download/3.1.6.2/pandoc-3.1.6.2-linux-amd64.tar.gz && \
```
Comment (gitstashpop) [inline Dockerfile:6]: Can you update your commit message to explain why this change was needed?

**Review by gitstashpop — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 4]
```diff
@@ -1,8 +1,12 @@
 FROM public.ecr.aws/amazonlinux/amazonlinux:2022
 
 RUN yum -y update
-RUN yum install -y tar net-tools curl vim unzip less libevent-devel openssl-devel python-devel libtool git patch make gcc --allowerasing
-RUN git clone https://github.com/pgbouncer/pgbouncer.git --branch "pgbouncer_1_15_0" && \
+RUN yum install -y wget && \
+    yum install -y tar net-tools curl vim unzip less libevent-devel openssl-devel python-devel libtool git patch make gcc --allowerasing
```
Comment (gitstashpop) [inline Dockerfile:5]: just append `wget` after gcc. It doesn't need to be a separate call to `yum install`

**Review by gitstashpop — APPROVED** [review]

File: Dockerfile [hunk 5]
```diff
@@ -1,16 +1,23 @@
...
+#RUN  apt-get update \
+#  && apt-get install -y wget \
+#  && rm -rf /var/lib/apt/lists/*
 FROM public.ecr.aws/amazonlinux/amazonlinux:2022
 
 RUN yum -y update
+RUN yum install -y wget
 RUN yum install -y tar net-tools curl vim unzip less libevent-devel openssl-devel python-devel libtool git patch make gcc --allowerasing
-RUN git clone https://github.com/pgbouncer/pgbouncer.git --branch "pgbouncer_1_15_0" && \
+RUN wget https://github.com/jgm/pandoc/releases/download/3.1.6.2/pandoc-3.1.6.2-linux-amd64.tar.gz && \
```
Comment (yahavb) [inline Dockerfile:6]: is there a cpu-platform agnostic option? i.e. no x86 or amd64 notation. I want us to think about multi-arch image that supports graviton and x86

File: Dockerfile [hunk 6]
```diff
@@ -1,16 +1,23 @@
+#FROM ubuntu:14.04
+#RUN  apt-get update \
+#  && apt-get install -y wget \
+#  && rm -rf /var/lib/apt/lists/*
 FROM public.ecr.aws/amazonlinux/amazonlinux:2022
 
 RUN yum -y update
+RUN yum install -y wget
 RUN yum install -y tar net-tools curl vim unzip less libevent-devel openssl-devel python-devel libtool git patch make gcc --allowerasing
-RUN git clone https://github.com/pgbouncer/pgbouncer.git --branch "pgbouncer_1_15_0" && \
```
Comment (yahavb) [inline Dockerfile:5]: Can we pls add the --branch value as parameter? Something like https://github.com/aws-samples/containerized-game-servers/blob/master/lyra/server/assets-image-multiarch/Dockerfile.template

**Review by yahavb — COMMENTED** [review]
few questions that can help the build more flexible
